node/core/dispute-coordinator/src/tests.rs

```diff
@@ -3193,3 +3193,134 @@ fn participation_for_included_candidates() {
 		})
 	});
 }
+
+/// Shows that importing backing votes when a backing event is being processed
```

Comment:
:+1:

Reply:
Nice, would be even better if the import happened due to chain scraping, but that is just a nit (haven't checked how easy that would be). Adding a comment that usually the import would happen via chain scraping would be good though.